app/app.go

```diff
@@ -700,7 +734,8 @@ func New(
 		capabilitytypes.ModuleName, authtypes.ModuleName, banktypes.ModuleName, distrtypes.ModuleName,
 		stakingtypes.ModuleName, slashingtypes.ModuleName, govtypes.ModuleName, minttypes.ModuleName,
 		crisistypes.ModuleName, ibchost.ModuleName, genutiltypes.ModuleName, evidencetypes.ModuleName,
-		authz.ModuleName, ibctransfertypes.ModuleName, // icatypes.ModuleName,
+		authz.ModuleName,
+		ibctransfertypes.ModuleName, ibchost.ModuleName, icatypes.ModuleName, // ibcfeetypes.ModuleName
```

Comment:
Same here, It is already added above https://github.com/umee-network/umee/pull/1963/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R736